src/modules/infobox/InfoboxModule.vue

```diff
@@ -40,7 +41,9 @@ const showContainer = computed(() => {
             (showElevationProfile.value && showFeatureInfoInTooltip.value))
     )
 })
-const showTooltipToggle = computed(() => showFeatureInfoInBottomPanel.value)
+const showTooltipToggle = computed(
+    () => showFeatureInfoInBottomPanel.value && width.value >= MAX_WIDTH_SHOW_FLOATING_TOOLTIP
+)
```

Comment:
That's weird but if I resize my viewport to bigger than the threshold and then back to mobile, the floating toogle button stays there (as if this wasn't re-computed)

Reply:
I can't seem to manage to reproduce it and it behaves as expected locally (and on sys dev). We can take a quick look tomorrow to see if we can reproduce this on your device and check what's the problem.